src/courtroom.cpp

```diff
@@ -4090,6 +4098,7 @@ bool Courtroom::effects_dropdown_find_and_set(QString effect)
 
 QString Courtroom::get_char_sfx()
 {
+  if (!ui_sfx->isChecked()) return "0";
```

Comment:
```suggestion
  if (!ui_sfx->isChecked())
    return "1";
```
Consistent with default return value of [`get_sfx_name`](https://github.com/AttorneyOnline/AO2-Client/blob/849f91d991bf0e95d579df268e9ab358e09b1ac1/src/text_file_functions.cpp#L771).